### PR TITLE
Example folder- with required data sources and resources

### PR DESCRIPTION
**Pull request by meghana-pedduri**
Added example folder with provider, resources and data sources folder that would contain the resources and data sources

### CODE REVIEW

File: examples/data_sources/traceable_endpoint_id/data_source.tf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+data "traceable_endpoint_id" "endpoint" {
+  name="POST /Unauthenticated_Modification_of_external_APIs"
```
Comment (murari54) [inline examples/data_sources/traceable_endpoint_id/data_source.tf:2]: can we give some generic names here

File: examples/data_sources/traceable_service_id/data_source.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+data "traceable_service_id" "endpoint" {
+  service_name="nginx-automation-test"
```
Comment (murari54) [inline examples/data_sources/traceable_service_id/data_source.tf:2]: this is bit internal to our tests..use generic name s like test-service etc

File: examples/data_sources/traceable_syslog_integration/data_source.tf [hunk 3]
```diff
@@ -0,0 +1,7 @@
+data "traceable_syslog_integration" "syslog" {
+  name="prer-test"
+}
+
+data "traceable_splunk_integration" "splunk" {
+  name="aditya"
```
Comment (murari54) [inline examples/data_sources/traceable_syslog_integration/data_source.tf:6]: don't use name

File: examples/provider/provider.tf [hunk 4]
```diff
@@ -0,0 +1,4 @@
+provider "traceable" {
+  platform_url="https://api-dev.traceable.ai/graphql"
```
Comment (murari54) [inline examples/provider/provider.tf:2]: remove the URL

File: examples/provider/provider.tf [hunk 5]
```diff
@@ -0,0 +1,4 @@
...
+  api_token=jsondecode(data.aws_secretsmanager_secret_version.api_token.secret_string)["api_token"]
```
Comment (murari54) [inline examples/provider/provider.tf:3]: use generic secret variable

File: examples/data_sources/traceable_endpoint_id/data_source.tf (hunk 1 above)
Comment (meghana-pedduri) [reply examples/data_sources/traceable_endpoint_id/data_source.tf]: done

File: examples/data_sources/traceable_service_id/data_source.tf (hunk 2 above)
Comment (meghana-pedduri) [reply examples/data_sources/traceable_service_id/data_source.tf]: done

File: examples/data_sources/traceable_syslog_integration/data_source.tf (hunk 3 above)
Comment (meghana-pedduri) [reply examples/data_sources/traceable_syslog_integration/data_source.tf]: done

File: examples/provider/provider.tf (hunk 4 above)
Comment (meghana-pedduri) [reply examples/provider/provider.tf]: done

File: examples/provider/provider.tf (hunk 5 above)
Comment (meghana-pedduri) [reply examples/provider/provider.tf]: done

**Review by adityaKumar9006 — APPROVED** [review]

**Pull request merged**